python/solve.py

```diff
@@ -11,6 +11,7 @@
 from typing import Callable, Dict
 
 from instance import Instance
+from file_wrappers import StdoutFileWrapper
```

Comment:
`from file_wrappers import StdinFileWrapper, StdoutFileWrapper`

Reply:
File updated in latest commit